contents/swift/welcome-to-swift.md

```diff
@@ -76,6 +74,34 @@ To learn more about Optionals, such as Optional Chaining and "dangerously" Force
 
 If you would like to seek a second (or more) opinion about Optionals, check out this [StackOverflow answer](http://stackoverflow.com/questions/24003642/what-is-an-optional-value-in-swift).
 
+## Guard Statement
+
+Notice that the golden path in the code above is indented:
```

Comment:
It will be good to explain how `guard` statement works.
The code within a `guard` block is only executed if the condition is not true.

Maybe explicitly mention/show that `path` can be used outside the indented block but it doesn't work for `if`?

Also, you can mention how guard clauses are often use to enforce the pre-conditions of a method (related to fail fast and defensive programming)

Reply:
also to avoid pyramid of doom (nested if-if-if-if...)